external/gtest.cmake

```diff
@@ -35,8 +35,8 @@
 # VS 2015     14           1900            std::tuple
 # VS 2017     15           >= 1910         std::tuple
 
-if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/googletest.tar.gz)
-    set(URL ${REPOSITORY_DIR}/build/ThirdParty/share/googletest.tar.gz)
+if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/googletest-release-1.8.1.tar.gz)
```

Comment:
What I was trying to do was make the override a little less restrictive by not requiring the release in the name.  But I must have done something wrong...sorry I broke it.

We can leave that for a later PR after the dust settles :)

Reply:
ok, thanks. Yes, I like the idea of the simpler manual names for air-gapped.